Sources/Profile-Modify.php

```diff
@@ -18,6 +18,8 @@
 if (!defined('SMF'))
 	die('No direct access...');
 
+const MAX_EXTERNAL_URL_SIZE = 255;
```

Comment:
I wonder whether it would be better to retrieve the maximum length of the field from the database rather than defining a constant. That way, if the length of the field ever changes, the function will still use the correct value.

Reply:
I thought about it but the amount of times this length will be changed is quite low and thus it doesn't justify to do a query to the dababase :/

We can always increase the field length on update to a more reasonable value

Reply:
Fair point. Still, instead of a constant, maybe we should make this a variable and declare it somewhere that will allow mods to change it via hooks instead of file edits.

Reply:
yep, that can be done yes.